static_src/components/users.jsx

```diff
@@ -23,6 +23,7 @@ const ORG_MANAGER = 'org_manager';
 const SPACE_MANAGER = 'space_manager';
 const ORG_ENTITY = 'organization';
 const SPACE_ENTITY = 'space';
+const cfAPILink = 'https://cloud.gov/docs/getting-started/setup/#set-up-the-command-line';
```

Comment:
let's change this to https://docs.cloudfoundry.org/adminguide/cli-user-management.html#space-roles